homeassistant/components/xiaomi_miio/air_quality.py

```diff
@@ -32,35 +26,20 @@
 
 PROP_TO_ATTR = {
     "carbon_dioxide_equivalent": ATTR_CO2E,
```

Comment:
Is this something else than carbon dioxide level?

https://developers.home-assistant.io/docs/en/entity_air_quality.html#properties

Reply:
I think so but I'm not an expert. ;-) https://www.google.com/search?q=co2e+vs+co2

Reply:
The funny thing here is: yes, CO2e is related but different (read more [here](https://www.sustainablebusinesstoolkit.com/difference-between-co2-and-co2e/). The b1 API returns this name/value. However, at the display of the b1 monitor, the value is shown as CO2a! The explanetion on the monitor says it cannot measure the value directly, so it calculated the value (Approximation) . 
Perhaps we should stick with CO2!